instance/static/html/instance/appserver.html

```diff
@@ -47,11 +47,18 @@
     </table>
 
     <button ng-disabled="is_active || !appserver.is_healthy"
-            tooltip="This will update the DNS entries so that this app server is used for the instance. Only one app server can be active at any time."
-            tooltip-placement="right"
+            tooltip="This will add this app server to the load balancer pool for this instance."
+            tooltip-placement="below"
             ng-click="make_appserver_active()">
       Activate this app server
     </button>
+
+    <button ng-disabled="!is_active || instance.active_appservers.length < 2"
```

Comment:
@smarnach  Yep, and I think we should use `instance.shut_down()` anyway, so that the monitoring is removed.  That's worth having as a button on the instance, with a big "ARE YOU SURE?" alert.